cmd/archeio/main.go

```diff
@@ -77,11 +138,14 @@ func handler(w http.ResponseWriter, r *http.Request) {
 	path := r.URL.Path
 	switch {
 	case strings.HasPrefix(path, "/v2/"):
+		serverMetrics.Requests.With(prometheus.Labels{"code": fmt.Sprintf("%v", http.StatusPermanentRedirect), "method": "GET", "version": "v2"}).Inc()
 		doV2(w, r)
 	case strings.HasPrefix(path, "/v1/"):
+		serverMetrics.Requests.With(prometheus.Labels{"code": fmt.Sprintf("%v", http.StatusPermanentRedirect), "method": "GET", "version": "v2"}).Inc()
```

Comment:
is this always a GET?

Reply:
Whoops, that shouldn't be like that. Fixed in 3599385